framework/jaif-inference-scripts/infer-and-annotate.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+# This script receives as arguments:
+# 1. Processor name.
+# 2. Classpath (target project's classpath and annotation-file-utilities.jar path)
+# 3. Project source folder path
+# 4. Extra processor arguments
+
+# Example of usage:
+#./infer-and-annotate.sh org.checkerframework.common.value.ValueChecker $PROJECT_CP:..../annotation-file-utilities.jar project_src_folder/ $EXTRA_PROCESSOR_ARGS
+# $PROJECT_CP must contain a path to annotation-file-utilities.jar. In case of 
```

Comment:
In the line above, annotation-file-utilities.jar is specified in addition to PROJECT_CP, but this line states that annotation-file-utilities.jar must be in PROJECT_CP as well.  Why specify it twice?